rust/ql/test/query-tests/security/CWE-825/lifetime.rs

```diff
@@ -651,20 +651,20 @@ pub fn test_implicit_derefs() {
 		let str2;
 		{
 			let str1 = "bar";
-			str2 = "foo".to_string() + &str1; // $ Source[rust/access-after-lifetime-ended]=str1
+			str2 = "foo".to_string() + &str1;
```

Comment:
@geoffw0 : `&str1` is no longer considered a source because of [this](https://github.com/github/codeql/blob/52abf3ba02242488bbf3e053b2f65ffe82e79a72/rust/ql/lib/codeql/rust/security/AccessAfterLifetimeExtensions.qll#L77) restriction.

Reply:
I think it actually _is_ still considered a source, but the results are excluded due to the restriction you linked to.  And these inline expectations only show sources associated with results, as I understand it, hence it is now hidden.

In any case both results we lost here are spurious, so I'm happy with this.